src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -260,6 +260,7 @@ void ConnectivityManagerImpl::DriveStationState()
         // Ensure that the WFX is started.
         if ((serr = wfx_wifi_start()) != SL_STATUS_OK)
         {
+            err = CHIP_ERROR_INTERNAL;
```

Comment:
But why ? Shouldn't we translate `serr` to `CHIP_ERROR` before logging it ? what value do we have in logging an hard_coded error code ?

Reply:
Removed the unused error variable